Sources/Sandbox.Game/Game/Entities/MyEntity.cs

```diff
@@ -100,6 +100,13 @@ public MyGameLogicComponent GameLogic
             set { Components.Add<MyGameLogicComponent>(value); }
         }
 
+		private HashSet<MyGameLogicComponent> m_gameLogicSet = null;
```

Comment:
I thought about delegates, but then felt that would just be inconsistent with how logic is currently attached, and a bit confusing, while also forcing mods to change (a bad thing).